workflows/wdl/iss_published/recipe.py

```diff
@@ -50,17 +50,24 @@ def process_fov(field_num: int, experiment_str: str):
     filt.run(dots, verbose=True, in_place=True)
     filt.run(nuclei, verbose=True, in_place=True)
 
-    print("Detecting")
-    detector = DetectSpots.BlobDetector(
+    print("Finding Spots")
+    bd = FindSpots.BlobDetector(
         min_sigma=1,
         max_sigma=10,
         num_sigma=30,
         threshold=0.01,
         measurement_type='mean',
     )
 
-    intensities = detector.run(filtered_imgs, blobs_image=dots, blobs_axes=(Axes.ROUND, Axes.ZPLANE))
+    dots_max_projector = Filter.Reduce((Axes.ROUND, Axes.ZPLANE), func="max",
+                                       module=Filter.Reduce.FunctionSource.np)
+    dots_max = dots_max_projector.run(dots)
```

Comment:
I was going to suggest that this use the ImageStack.reduce method, but I guess that hasn't landed.